src/modules/pos/validator/components/ChangeCommission/Summary/index.js

```diff
@@ -0,0 +1,11 @@
+/* istanbul ignore file */
+import { connect } from 'react-redux';
+import { validatorRegistered } from 'src/modules/pos/validator/store/actions/validator';
+import ChangeCommissionSummary from './ChangeCommissionSummary';
+
+
+const mapDispatchToProps = {
+  validatorRegistered,
+};
+
+export default connect(() => undefined, mapDispatchToProps)(ChangeCommissionSummary);
```

Comment:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/8784876/218067941-e9c7745f-fa59-4598-9165-7abe605498a4.png">

To prevent this warning we need to:
`connect(null, mapDispatchToProps)(ChangeCommissionSummary);`

Or just use dispatch directly in the component `ChangeCommissionSummary`  and not using a connect wrapper